diff -urN gnucash-1.6.5.orig/configure.in gnucash-1.6.5/configure.in
--- gnucash-1.6.5.orig/configure.in	Wed Mar  6 00:06:19 2002
+++ gnucash-1.6.5/configure.in	Wed Mar  6 00:21:17 2002
@@ -403,7 +403,9 @@
 
 ### --------------------------------------------------------------------------
 ### Gnome XML -- GNOME_XML_LIB is define by GNOME_XML_CHECK
-GNOME_XML_CFLAGS=`$GNOME_CONFIG --cflags xml`
+#GNOME_XML_CFLAGS=`$GNOME_CONFIG --cflags xml`
+# This should be checked if it is done The Right Way (tm).
+GNOME_XML_CFLAGS="-I/usr/X11R6/include"
 
 AC_SUBST(GNOME_XML_CFLAGS)
 
