--- gnucash-1.6.5/src/gnome/Makefile.am~	Tue Feb 12 20:22:02 2002
+++ gnucash-1.6.5/src/gnome/Makefile.am	Tue Feb 12 20:22:02 2002
@@ -175,7 +175,7 @@
 	    -e 's:@-GNC_PIXMAP_DIR-@:${GNC_PIXMAP_DIR}:g' \
 	    -e 's:@-GNC_ACCOUNTS_DIR-@:${GNC_ACCOUNTS_DIR}:g'
 	mv $@.tmp $@
-BUILT_SOURCES += gnc-dir.h
+BUILT_SOURCES = gnc-dir.h
 
 # We have to do this because otherwise automake insists on putting
 # these files into the dist tarfile.  If there's a a better way,
--- gnucash-1.6.5/src/guile/Makefile.am~	Tue Feb 12 20:22:58 2002
+++ gnucash-1.6.5/src/guile/Makefile.am	Tue Feb 12 20:22:58 2002
@@ -63,8 +63,8 @@
           "(set! %load-path (cons \"${G_WRAP_MODULE_DIR}\" %load-path)) \
            (primitive-load \"./gnc.gwp\") \
            (gw:generate-module \"gnc\")"
-BUILT_SOURCES += gnc.h gnc.c
-CLEANFILES += gnc.h gnc.c gnc.html
+BUILT_SOURCES = gnc.h gnc.c
+CLEANFILES = gnc.h gnc.c gnc.html
 
 ## We borrow guile's convention and use @-...-@ as the substitution
 ## brackets here, instead of the usual @...@.  This prevents autoconf
--- gnucash-1.6.5/src/quotes/Makefile.am~	Tue Feb 12 20:23:33 2002
+++ gnucash-1.6.5/src/quotes/Makefile.am	Tue Feb 12 20:23:33 2002
@@ -23,7 +23,7 @@
             -e 's:@-PERLINCL-@:${PERLINCL}:g'
 	chmod +x $@.tmp
 	mv $@.tmp $@
-CLEANFILES += finance-quote-helper
+CLEANFILES = finance-quote-helper
 
 update-finance-quote: update-finance-quote.in
 	rm -f $@.tmp
--- gnucash-1.6.5/src/scm/Makefile.am~	Tue Feb 12 20:24:18 2002
+++ gnucash-1.6.5/src/scm/Makefile.am	Tue Feb 12 20:24:18 2002
@@ -72,7 +72,7 @@
 guile-strings.c: ${gncscm_DATA}
 	rm -f guile-strings.c
 	guile -s ./xgettext.scm ${gncscm_DATA}
-CLEANFILES += guile-strings.c
+CLEANFILES = guile-strings.c
 
 all-local: guile-strings.c
 
--- gnucash-1.6.5/src/scm/gnumeric/Makefile.am~	Tue Feb 12 20:25:02 2002
+++ gnucash-1.6.5/src/scm/gnumeric/Makefile.am	Tue Feb 12 20:25:02 2002
@@ -12,6 +12,6 @@
 guile-strings.c: ${gncscm_DATA}
 	rm -f guile-strings.c
 	guile -s ../xgettext.scm ${gncscm_DATA}
-CLEANFILES += guile-strings.c
+CLEANFILES = guile-strings.c
 
 all-local: guile-strings.c
--- gnucash-1.6.5/src/scm/printing/Makefile.am~	Tue Feb 12 20:26:11 2002
+++ gnucash-1.6.5/src/scm/printing/Makefile.am	Tue Feb 12 20:26:11 2002
@@ -12,6 +12,6 @@
 guile-strings.c: ${gncscm_DATA}
 	rm -f guile-strings.c
 	guile -s ../xgettext.scm ${gncscm_DATA}
-CLEANFILES += guile-strings.c
+CLEANFILES = guile-strings.c
 
 all-local: guile-strings.c
--- gnucash-1.6.5/src/scm/qif-import/Makefile.am~	Tue Feb 12 20:26:44 2002
+++ gnucash-1.6.5/src/scm/qif-import/Makefile.am	Tue Feb 12 20:26:44 2002
@@ -21,6 +21,6 @@
 guile-strings.c: ${gncscm_DATA}
 	rm -f guile-strings.c
 	guile -s ../xgettext.scm ${gncscm_DATA}
-CLEANFILES += guile-strings.c
+CLEANFILES = guile-strings.c
 
 all-local: guile-strings.c
--- gnucash-1.6.5/src/scm/report/Makefile.am~	Tue Feb 12 20:27:22 2002
+++ gnucash-1.6.5/src/scm/report/Makefile.am	Tue Feb 12 20:27:22 2002
@@ -37,6 +37,6 @@
 guile-strings.c: ${SCM_FILES}
 	rm -f guile-strings.c
 	guile -s ../xgettext.scm ${SCM_FILES}
-CLEANFILES += guile-strings.c
+CLEANFILES = guile-strings.c
 
 all-local: guile-strings.c
--- gnucash-1.6.5/Makefile.am~	Tue Feb 12 20:28:49 2002
+++ gnucash-1.6.5/Makefile.am	Tue Feb 12 20:28:49 2002
@@ -56,7 +56,7 @@
 	chmod +x $@.tmp
 	mv $@.tmp $@
 
-DISTCLEANFILES += \
+DISTCLEANFILES = \
   TAGS.stamp cscope.files cscope.out etags.files make-gnucash-patch
 
 cscope.files:
--- gnucash-1.6.5/doc-tools/Makefile.am~	Tue Feb 12 20:29:26 2002
+++ gnucash-1.6.5/doc-tools/Makefile.am	Tue Feb 12 20:29:26 2002
@@ -25,4 +25,4 @@
 	chmod +x $@.tmp
 	mv $@.tmp $@
 
-CLEANFILES += build-help-index
+CLEANFILES = build-help-index
--- gnucash-1.6.5/accounts/C/Makefile.am~	Tue Feb 12 20:31:30 2002
+++ gnucash-1.6.5/accounts/C/Makefile.am	Tue Feb 12 20:31:30 2002
@@ -36,7 +36,3 @@
   acctlist_retiremt.gnucash-xea \
   acctlist_spouseinc.gnucash-xea \
   acctlist_spouseretire.gnucash-xea
-
-EXTRA_DIST = \
-  ${account_DATA} \
-  .cvsignore
--- gnucash-1.6.5/rpm/Makefile.am~	Tue Feb 12 20:38:39 2002
+++ gnucash-1.6.5/rpm/Makefile.am	Tue Feb 12 20:38:39 2002
@@ -15,4 +15,4 @@
             -e 's:@-VERSION-@:${VERSION}:'
 	mv $@.tmp $@
 
-CLEANFILES += gnucash.spec
+CLEANFILES = gnucash.spec
--- gnucash-1.6.5/src/experimental/cgi-bin/Makefile.am~	Tue Feb 12 20:39:51 2002
+++ gnucash-1.6.5/src/experimental/cgi-bin/Makefile.am	Tue Feb 12 20:39:51 2002
@@ -3,14 +3,10 @@
 
 bin_PROGRAMS = hello hello2 fastcgi-hello hello3 gnc-server
 
-CFLAGS = @CFLAGS@ ${GLIB_CFLAGS} 
-
 LDADD = 			\
   ../../engine/libgncengine.la 	\
   -lxml -lghttp -lglib -lfcgi
 
-INCLUDES =
-
 hello_SOURCES = \
   hello.c
 
@@ -28,14 +24,11 @@
 
 noinst_HEADERS = 
 
-
 EXTRA_DIST = \
-  .cvsignore \
   README 
 
-CFLAGS = @CFLAGS@ 
-
 #  glib puts its header files in a bizarre place ...
 INCLUDES = 			\
   -I/usr/lib/glib/include 	\
-  -I../../engine 
+  -I../../engine 		\
+  ${GLIB_CFLAGS}
--- gnucash-1.6.5/configure.in~	Tue Feb 12 20:52:38 2002
+++ gnucash-1.6.5/configure.in	Tue Feb 12 20:52:38 2002
@@ -670,8 +670,6 @@
           src/gnome/Makefile
           src/guile/Makefile
           src/optional/Makefile
-          src/optional/swig/Makefile
-          src/optional/swig/examples/Makefile
           src/pixmaps/Makefile
           src/quotes/Makefile
           src/register/Makefile
