--- gnucash-2.6.3/src/core-utils/gnc-path.c.orig	2014-03-01 20:55:35.000000000 +0100
+++ gnucash-2.6.3/src/core-utils/gnc-path.c	2014-06-24 21:56:28.519968927 +0200
@@ -23,12 +23,11 @@
 #include "gnc-path.h"
 #include "gncla-dir.h"
 #include <stdio.h>
-#include "binreloc.h"
 
 gchar *gnc_path_get_prefix()
 {
     //printf("Returning prefix %s\n", gnc_gbr_find_prefix (PREFIX));
-    return gnc_gbr_find_prefix (PREFIX);
+    return g_strdup (PREFIX);
 }
 
 /** Returns the bindir path, usually
@@ -38,7 +37,7 @@
 gchar *gnc_path_get_bindir()
 {
     //printf("Returning bindir %s\n", gnc_gbr_find_bin_dir (BINDIR));
-    return gnc_gbr_find_bin_dir (BINDIR);
+    return g_strdup (BINDIR);
 }
 
 /** Returns the libdir path, usually
@@ -48,7 +47,7 @@
 gchar *gnc_path_get_libdir()
 {
     //printf("Returning libdir %s\n", gnc_gbr_find_lib_dir (LIBDIR));
-    return gnc_gbr_find_lib_dir (LIBDIR);
+    return g_strdup (LIBDIR);
 }
 
 /** Returns the datadir path, usually
@@ -57,7 +56,7 @@
  * @returns A newly allocated string. */
 gchar *gnc_path_get_pkgdatadir()
 {
-    gchar *datadir = gnc_gbr_find_data_dir (DATADIR);
+    gchar *datadir = g_strdup (DATADIR);
     gchar *result = g_build_filename (datadir, "gnucash", (char*)NULL);
     g_free (datadir);
     //printf("Returning pkgdatadir %s\n", result);
@@ -70,7 +69,7 @@
  * @returns A newly allocated string. */
 gchar *gnc_path_get_pkgdocdir()
 {
-    gchar *docdir = gnc_gbr_find_data_dir (DATADIR);
+    gchar *docdir = g_strdup (DATADIR);
     gchar *result = g_build_filename (docdir, "doc", "gnucash", (char*)NULL);
     g_free (docdir);
     //printf("Returning pkgdocdir %s\n", result);
@@ -83,7 +82,7 @@
  * @returns A newly allocated string. */
 gchar *gnc_path_get_pkgsysconfdir()
 {
-    gchar *sysconfdir = gnc_gbr_find_etc_dir (SYSCONFDIR);
+    gchar *sysconfdir = g_strdup (SYSCONFDIR);
     gchar *result = g_build_filename (sysconfdir, "gnucash", (char*)NULL);
     g_free (sysconfdir);
     //printf("Returning pkgsysconfdir %s\n", result);
