--- gnucash-1.8.11/src/engine/gnc-engine-util.c.orig	2003-04-12 19:02:55.000000000 +0200
+++ gnucash-1.8.11/src/engine/gnc-engine-util.c	2005-03-28 12:45:41.069095667 +0200
@@ -234,7 +234,7 @@
   if (!fout) fout = stderr;
 
   fprintf (fout, "Clock %d Elapsed: %ld.%06lds %s: ",
-           clockno, now.tv_sec, now.tv_usec, prettify (function_name));
+           clockno, (long)now.tv_sec, (long)now.tv_usec, prettify (function_name));
 
   va_start (ap, format);
 
@@ -265,8 +265,8 @@
 
   fprintf (fout, "Clock %d Total Elapsed: %ld.%06lds  %s: ",
            clockno,
-           gnc_clock_total[clockno].tv_sec,
-           gnc_clock_total[clockno].tv_usec,
+           (long)gnc_clock_total[clockno].tv_sec,
+           (long)gnc_clock_total[clockno].tv_usec,
            prettify (function_name));
 
   va_start (ap, format);
--- gnucash-1.8.11/src/backend/file/test/test-xml-transaction.c.orig	2002-12-28 22:25:52.000000000 +0100
+++ gnucash-1.8.11/src/backend/file/test/test-xml-transaction.c	2005-03-28 12:44:25.393385305 +0200
@@ -130,8 +130,8 @@
             {
                 g_free(num);
                 return g_strdup_printf ("values differ: %lld/%lld v %lld/%lld",
-					(*num).num, (*num).denom,
-					val.num, val.denom);
+					(long long)(*num).num, (long long)(*num).denom,
+					(long long)val.num, (long long)val.denom);
             }
             g_free(num);
         }
@@ -142,15 +142,15 @@
 
             if (!gnc_numeric_equal(*num, val)) {
               return g_strdup_printf( "quantities differ under _equal: %lld/%lld v %lld/%lld",
-                                      (*num).num, (*num).denom,
-                                      val.num, val.denom );
+                                      (long long)(*num).num, (long long)(*num).denom,
+                                      (long long)val.num, (long long)val.denom );
             }
             if(!gnc_numeric_equal(*num, val))
             {
                 g_free(num);
                 return g_strdup_printf ("quantities differ: %lld/%lld v %lld/%lld",
-					(*num).num, (*num).denom,
-					val.num, val.denom);
+					(long long)(*num).num, (long long)(*num).denom,
+					(long long)val.num, (long long)val.denom);
             }
             g_free(num);
         }
--- gnucash-1.8.11/src/gnome-utils/dialog-account.c.orig	2004-12-12 00:04:27.000000000 +0100
+++ gnucash-1.8.11/src/gnome-utils/dialog-account.c	2005-03-28 12:50:48.534942768 +0200
@@ -1325,7 +1325,7 @@
 gnc_type_list_row_set_active(GtkCList *type_list, gint type, gboolean state)
 {
   GtkStyle *style = gtk_widget_get_style(GTK_WIDGET(type_list));
-  gint row = gtk_clist_find_row_from_data(type_list, (gpointer)type);
+  gint row = gtk_clist_find_row_from_data(type_list, (gpointer)(size_t)type);
 
   if (state)
   {
--- gnucash-1.8.11/src/gnome-utils/gnc-html.c.orig	2003-04-12 19:03:30.000000000 +0200
+++ gnucash-1.8.11/src/gnome-utils/gnc-html.c	2005-03-28 12:53:40.313578949 +0200
@@ -1315,7 +1315,7 @@
 static gboolean 
 raw_html_receiver (gpointer     engine,
                    const gchar *data,
-                   guint        len,
+                   size_t       len,
                    gpointer     user_data)
 {
   FILE *fh = (FILE *) user_data;
--- gnucash-1.8.11/src/gnome/window-acct-tree.c.orig	2003-04-29 17:28:05.000000000 +0200
+++ gnucash-1.8.11/src/gnome/window-acct-tree.c	2005-03-28 12:59:13.647224083 +0200
@@ -95,7 +95,7 @@
                                     gboolean sensitive)
 {
   g_list_foreach(win->account_sensitives, (GFunc)gtk_widget_set_sensitive,
-		 (gpointer)sensitive);
+		 (gpointer)(size_t)sensitive);
 }
 
 
@@ -421,7 +421,7 @@
     }
   }
 
-  return (gpointer)(helper_res->has_splits || helper_res->has_ro_splits);
+  return (gpointer)(size_t)(helper_res->has_splits || helper_res->has_ro_splits);
 }
 
 static void
--- gnucash-1.8.11/src/gnome/dialog-sxsincelast.c.orig	2004-12-12 00:04:26.000000000 +0100
+++ gnucash-1.8.11/src/gnome/dialog-sxsincelast.c	2005-03-28 12:58:11.675155486 +0200
@@ -2482,7 +2482,7 @@
                                 
                                 /* No ... we reallly need to skip the entire
                                    transaction, at this point. :( Bug#130330. */
-                                PERR( "err transaction %.8x", (int)newT );
+                                PERR( "err transaction %.8x", (int)(size_t)newT /* XXX */);
                                 errFlag = TRUE;
                                 break;
                         }
--- gnucash-1.8.11/src/business/business-core/file/gnc-bill-term-xml-v2.c.orig	2003-09-11 19:45:22.000000000 +0200
+++ gnucash-1.8.11/src/business/business-core/file/gnc-bill-term-xml-v2.c	2005-03-28 13:01:02.824926611 +0200
@@ -669,7 +669,7 @@
   if (count != gncBillTermGetRefcount(term) && !gncBillTermGetInvisible(term)) {
     PWARN("Fixing refcount on billterm %s (%lld -> %d)\n",
 	  guid_to_string(gncBillTermGetGUID(term)),
-	  gncBillTermGetRefcount(term), count)
+	  (long long)gncBillTermGetRefcount(term), count)
       gncBillTermSetRefcount(term, count);
   }
 }
--- gnucash-1.8.11/src/business/business-core/file/gnc-tax-table-xml-v2.c.orig	2003-09-11 19:45:27.000000000 +0200
+++ gnucash-1.8.11/src/business/business-core/file/gnc-tax-table-xml-v2.c	2005-03-28 13:02:14.453678325 +0200
@@ -620,7 +620,7 @@
   if (count != gncTaxTableGetRefcount(table) && !gncTaxTableGetInvisible(table)) {
     PWARN("Fixing refcount on taxtable %s (%lld -> %d)\n",
 	  guid_to_string(gncTaxTableGetGUID(table)),
-	  gncTaxTableGetRefcount(table), count)
+	  (long long)gncTaxTableGetRefcount(table), count)
       gncTaxTableSetRefcount(table, count);
   }
 }
--- gnucash-1.8.11/src/business/business-gnome/dialog-customer.c.orig	2003-06-16 01:05:20.000000000 +0200
+++ gnucash-1.8.11/src/business/business-gnome/dialog-customer.c	2005-03-28 13:03:01.319394930 +0200
@@ -265,7 +265,7 @@
   if (safe_strcmp (gtk_entry_get_text (GTK_ENTRY (cw->id_entry)), "") == 0) {
     gtk_entry_set_text (GTK_ENTRY (cw->id_entry),
 			g_strdup_printf ("%.6lld",
-					 gncCustomerNextID (cw->book)));
+					 (long long)gncCustomerNextID (cw->book)));
   }
 
   /* Now save it off */
--- gnucash-1.8.11/src/business/business-gnome/dialog-employee.c.orig	2003-06-16 01:05:21.000000000 +0200
+++ gnucash-1.8.11/src/business/business-gnome/dialog-employee.c	2005-03-28 13:04:08.291273260 +0200
@@ -209,7 +209,7 @@
   if (safe_strcmp (gtk_entry_get_text (GTK_ENTRY (ew->id_entry)), "") == 0) {
     gtk_entry_set_text (GTK_ENTRY (ew->id_entry),
 			g_strdup_printf ("%.6lld",
-					 gncEmployeeNextID (ew->book)));
+					 (long long)gncEmployeeNextID (ew->book)));
   }
 
   /* Now save it off */
--- gnucash-1.8.11/src/business/business-gnome/dialog-job.c.orig	2003-04-21 04:42:07.000000000 +0200
+++ gnucash-1.8.11/src/business/business-gnome/dialog-job.c	2005-03-28 13:05:10.387327778 +0200
@@ -119,7 +119,7 @@
   res = gtk_entry_get_text (GTK_ENTRY (jw->id_entry));
   if (safe_strcmp (res, "") == 0) {
     gtk_entry_set_text (GTK_ENTRY (jw->id_entry),
-			g_strdup_printf ("%.6lld", gncJobNextID(jw->book)));
+			g_strdup_printf ("%.6lld", (long long)gncJobNextID(jw->book)));
   }
 
   /* Now save it off */
--- gnucash-1.8.11/src/business/business-gnome/dialog-invoice.c.orig	2004-12-12 00:04:25.000000000 +0100
+++ gnucash-1.8.11/src/business/business-gnome/dialog-invoice.c	2005-03-28 13:04:42.058145852 +0200
@@ -297,7 +297,7 @@
   res = gtk_entry_get_text (GTK_ENTRY (iw->id_entry));
   if (safe_strcmp (res, "") == 0) {
   gtk_entry_set_text (GTK_ENTRY (iw->id_entry),
-		      g_strdup_printf ("%.6lld", gncInvoiceNextID(iw->book)));
+		      g_strdup_printf ("%.6lld", (long long)gncInvoiceNextID(iw->book)));
   }
       
   return TRUE;
--- gnucash-1.8.11/src/business/business-gnome/dialog-order.c.orig	2003-01-19 00:36:36.000000000 +0100
+++ gnucash-1.8.11/src/business/business-gnome/dialog-order.c	2005-03-28 13:05:37.406825339 +0200
@@ -714,7 +714,7 @@
   /* Setup initial values */
   ow->order_guid = *gncOrderGetGUID (order);
   gtk_entry_set_text (GTK_ENTRY (ow->id_entry),
-		      g_strdup_printf ("%.6lld", gncOrderNextID(bookp)));
+		      g_strdup_printf ("%.6lld", (long long)gncOrderNextID(bookp)));
       
   ow->component_id =
     gnc_register_gui_component (DIALOG_NEW_ORDER_CM_CLASS,
--- gnucash-1.8.11/src/business/business-gnome/dialog-vendor.c.orig	2004-12-12 00:04:25.000000000 +0100
+++ gnucash-1.8.11/src/business/business-gnome/dialog-vendor.c	2005-03-28 13:06:13.813064564 +0200
@@ -189,7 +189,7 @@
   if (safe_strcmp (gtk_entry_get_text (GTK_ENTRY (vw->id_entry)), "") == 0)
     gtk_entry_set_text (GTK_ENTRY (vw->id_entry),
 			g_strdup_printf ("%.6lld",
-					 gncVendorNextID(vw->book)));
+					 (long long)gncVendorNextID(vw->book)));
 
   /* Now save it off */
   {
--- gnucash-1.8.11/src/backend/postgres/putil.h.orig	2003-08-12 05:17:22.000000000 +0200
+++ gnucash-1.8.11/src/backend/postgres/putil.h	2005-03-28 17:57:31.854947120 +0200
@@ -318,7 +318,7 @@
 #define COMP_INT64(sqlname,fun,ndiffs) { 			\
    if (strtoll (DB_GET_VAL(sqlname,0), NULL, 0) != fun) {	\
       PINFO("mis-match: %s sql='%s', eng='%lld'", sqlname, 	\
-         DB_GET_VAL (sqlname,0), fun); 				\
+         DB_GET_VAL (sqlname,0), (long long)fun); 				\
       ndiffs++; 						\
    }								\
 }
--- gnucash-1.8.11/src/backend/postgres/table.m4.orig	2003-08-12 05:17:24.000000000 +0200
+++ gnucash-1.8.11/src/backend/postgres/table.m4	2005-03-28 18:01:48.376356492 +0200
@@ -365,7 +365,7 @@
    p = guid_to_string_buff (&(ptr->guid), p);
    p = stpcpy (p, "''`;");
    SEND_QUERY (be,be->buff, -1);
-   sql_version = (int) pgendGetResults (be, get_version_cb, (gpointer) -1);
+   sql_version = (int)(size_t) pgendGetResults (be, get_version_cb, (gpointer) -1);
 
    if (-1 == sql_version) return -1;
    return (sql_version - version_function($@));
@@ -395,7 +395,7 @@
    p = guid_to_string_buff (&(ptr->guid), p);
    p = stpcpy (p, "''` AND change = ''`d''`;");
    SEND_QUERY (be,be->buff, -1);
-   sql_version = (int) pgendGetResults (be, get_version_cb, (gpointer) -1);
+   sql_version = (int)(size_t) pgendGetResults (be, get_version_cb, (gpointer) -1);
 
    return sql_version;
 }
--- gnucash-1.8.11/src/backend/postgres/checkpoint.c.orig	2003-08-12 05:17:19.000000000 +0200
+++ gnucash-1.8.11/src/backend/postgres/checkpoint.c	2005-03-28 18:04:03.662874158 +0200
@@ -472,7 +472,7 @@
       gnc_timespec_to_iso8601_buff (chk.date_start, buf);
       PINFO("%s balance to %s baln=%lld/%lld clr=%lld/%lld rcn=%lld/%lld", 
         xaccAccountGetDescription (acc), buf,
-        b, deno, cl_b, deno, rec_b, deno);
+        (long long)b, (long long)deno, (long long)cl_b, (long long)deno, (long long)rec_b, (long long)deno);
       })
 
    /* add up loose entries since the checkpoint */
@@ -494,7 +494,7 @@
       gnc_timespec_to_iso8601_buff (as_of_date, buf);
       LEAVE("be=%p %s %s baln=%lld/%lld clr=%lld/%lld rcn=%lld/%lld", be, 
         xaccAccountGetDescription (acc), buf,
-        b, deno, cl_b, deno, rec_b, deno);
+        (long long)b, (long long)deno, (long long)cl_b, (long long)deno, (long long)rec_b, (long long)deno);
       })
 }
 
--- gnucash-1.8.11/src/backend/postgres/kvp-sql.c.orig	2003-02-13 03:59:20.000000000 +0100
+++ gnucash-1.8.11/src/backend/postgres/kvp-sql.c	2005-03-28 18:05:32.231608735 +0200
@@ -120,7 +120,7 @@
 ival_cb (PGBackend *be, PGresult *result, int j, gpointer data)
 {
   int ival = atoi (DB_GET_VAL ("ipath", 0));
-  return (gpointer) ival;
+  return (gpointer)(size_t) ival;
 }
 
 
@@ -142,7 +142,7 @@
    p = stpcpy (p, "';");
 
    SEND_QUERY (be,be->buff, 0);
-   ival = (int) pgendGetResults (be, ival_cb, (gpointer) 0);
+   ival = (int)(size_t) pgendGetResults (be, ival_cb, (gpointer) 0);
    if (ival) return ival;
 
    /* Else, this guid has never been stored before. 
--- gnucash-1.8.11/src/backend/postgres/price.c.orig	2003-07-28 06:00:26.000000000 +0200
+++ gnucash-1.8.11/src/backend/postgres/price.c	2005-03-28 18:06:12.541930183 +0200
@@ -200,7 +200,7 @@
 static gboolean
 commodity_mark_cb (gnc_commodity *cm, gpointer user_data)
 {
-   gint32 v = ((gint32) user_data) & 0xffff;
+   gint32 v = ((gint32)(size_t) user_data) & 0xffff;
    gnc_commodity_set_mark (cm, (gint16) v);
    return TRUE;
 }
--- gnucash-1.8.11/src/backend/postgres/upgrade.c.orig	2003-08-14 05:46:55.000000000 +0200
+++ gnucash-1.8.11/src/backend/postgres/upgrade.c	2005-03-28 18:06:56.019491164 +0200
@@ -126,7 +126,7 @@
 get_iguid_cb (PGBackend *be, PGresult *result, int j, gpointer data)
 {
    int fin = atoi(DB_GET_VAL ("iguid", j));
-   return (gpointer) fin;
+   return (gpointer)(size_t) fin;
 }
 
 
@@ -151,7 +151,7 @@
 
    p = "SELECT iguid FROM gncGUIDCache ORDER BY iguid DESC LIMIT 1;";
    SEND_QUERY (be,p, );
-   iguid = (guint32) pgendGetResults (be, get_iguid_cb, 0);
+   iguid = (guint32)(size_t) pgendGetResults (be, get_iguid_cb, 0);
    iguid ++;
 
    sprintf(buff, "CREATE SEQUENCE gnc_iguid_seq START %d;", iguid);
