--- gnucash-1.6.6/lib/srfi/srfi-2.scm~	Tue May 15 17:48:43 2001
+++ gnucash-1.6.6/lib/srfi/srfi-2.scm	Tue May 21 00:36:12 2002
@@ -22,7 +22,8 @@
 (cond
 
  ((or (string=? "1.3.4" (version))
-      (string=? "1.4" (version)))
+      (string=? "1.4" (version))
+      (string=? "1.4.1" (version)))
   (use-modules (ice-9 and-let*)))
 
  ((string=? "1.3" (version))
--- gnucash-1.6.6/lib/srfi/srfi-8.scm~	Tue May 15 17:48:43 2001
+++ gnucash-1.6.6/lib/srfi/srfi-8.scm	Tue May 21 00:36:28 2002
@@ -22,7 +22,8 @@
 (cond
  ((or (string=? "1.3" (version))
       (string=? "1.3.4" (version))
-      (string=? "1.4" (version)))
+      (string=? "1.4" (version))
+      (string=? "1.4.1" (version)))
   
   (use-modules (ice-9 slib))
   (require 'macro-by-example)
