--- gnucash-1.4.0/src/optional/swig/Makefile.am.orig	Wed Jun 14 10:05:40 2000
+++ gnucash-1.4.0/src/optional/swig/Makefile.am	Mon Jun 19 20:57:39 2000
@@ -67,11 +67,11 @@
 ## same as the .pm file, and automake won't tolerate libraries that
 ## aren't named lib*.so...
 install-data-local: libgncswig.la
-	cd ${perllibdir} && rm -f gnucash.so
-	if [ -f ${perllibdir}/libgncswig.so ]; then \
-		cd ${perllibdir} && ln -s libgncswig.so gnucash.so; \
-	elif [ -f ${perllibdir}/libgncswig.so.1.1 ]; then \
-		cd ${perllibdir} && ln -s libgncswig.so.1.1 gnucash.so; \
+	cd ${DESTDIR}/${perllibdir} && rm -f gnucash.so
+	if [ -f ${DESTDIR}/${perllibdir}/libgncswig.so ]; then \
+		cd ${DESTDIR}/${perllibdir} && ln -s libgncswig.so gnucash.so; \
+	elif [ -f ${DESTDIR}/${perllibdir}/libgncswig.so.1.1 ]; then \
+		cd ${DESTDIR}/${perllibdir} && ln -s libgncswig.so.1.1 gnucash.so; \
 	else \
 		echo "Can't find libgncswig shared library!" \
 		exit 1; \
