--- gnucash-1.6.5/src/doc/design/gnucash-design.texinfo~	Wed Feb 13 15:12:02 2002
+++ gnucash-1.6.5/src/doc/design/gnucash-design.texinfo	Wed Feb 13 15:12:02 2002
@@ -11,10 +11,10 @@
 @ifinfo
 @format
 
-@dircategory Programming
+@dircategory Programming Languages:
 
 @direntry
-* gnucash: (gnucash-design).       Design of the GnuCash program
+* gnucash: (gnucash-design).		Design of the GnuCash program
 @end direntry
 
 @end format
