--- gnucash-3.5/libgnucash/engine/qofbook.cpp.orig	2019-03-29 17:46:34.000000000 +0100
+++ gnucash-3.5/libgnucash/engine/qofbook.cpp	2019-04-06 09:39:22.820551177 +0200
@@ -615,12 +615,16 @@
     return col;
 }
 
+namespace {
+
 struct _iterate
 {
     QofCollectionForeachCB  fn;
     gpointer                data;
 };
 
+};
+
 static void
 foreach_cb (G_GNUC_UNUSED gpointer key, gpointer item, gpointer arg)
 {
--- gnucash-3.5/libgnucash/engine/qofid.cpp.orig	2019-01-12 00:57:38.000000000 +0100
+++ gnucash-3.5/libgnucash/engine/qofid.cpp	2019-04-06 09:38:29.207628926 +0200
@@ -303,12 +303,16 @@
 
 /* =============================================================== */
 
+namespace {
+
 struct _iterate
 {
     QofInstanceForeachCB      fcn;
     gpointer                data;
 };
 
+};
+
 static void
 foreach_cb (gpointer item, gpointer arg)
 {
